source/installation-guide/installing-wazuh-agent/linux/centos5/index.rst

```diff
@@ -18,3 +18,5 @@ The Wazuh agent is available for CentOS 5, and can be installed via package:
     :maxdepth: 0
 
     wazuh_agent_package_centos5
+
```

Comment:
What about those 2 new lines?

Reply:
I've researched about this change and I don't know the reason I made it, so discard it, please.